validator/engine/validator-in-browser.js

```diff
@@ -51,8 +51,8 @@ function getUrl(url) {
  */
 amp.validator.isAmpCacheUrl = function(url) {
   return (
-    url.toLowerCase().indexOf('cdn.ampproject.org') !== -1 ||
```

Comment:
Why these changes?

Reply:
After looking at #8424, I've reverted this change. We're going to have to ignore this alert unless there's a safer way of comparing the trailing part of a hostname.

/cc @honeybadgerdontcare